python/scope.py

```diff
@@ -1050,9 +1050,9 @@ async def eval_command(self, tree):
                 exit, value = await scope.eval_command(code)
                 if exit == "continue":
                     continue
+                if exit == "break":
+                    return False, None
```

Comment:
Awesome, `break` would work too

In the future, if we add labels, we can use the second item for labels